### ISSUE TITLE
Option to disable object brightness

### ISSUE THREAD

**Issue by RacerBG** [Stellarium/stellarium#1308]
**Is your feature request related to a problem? Please describe.**

My feature request is related to the artificial brightness (or glow) of the celestial objects while zooming in.

**Describe the solution you'd like**

I would like to suggest an option in the settings page of the planetarium to disable the bightness or the glow effect of the celestial objects (mainly planets and moons). For example when looking at Saturn with a telescope (depends on the magnification) you should see a clear disk and rings shape and no glow at all.

**Describe alternatives you've considered**

None.

**Additional context**

![Saturn](https://user-images.githubusercontent.com/68707872/94986753-f086c080-0569-11eb-87c3-cf3a9c68e496.png)

![Mars](https://user-images.githubusercontent.com/68707872/94986754-f11f5700-0569-11eb-83bd-f29ad90fd621.png)

Regards,
Georgi

**Comment by gzotti** [comment]
The "halo glow" vanishes automatically as you zoom in to get a clearer picture. Without this "halo" planets would vanish from view as they become too small. With too little magnification (3x plastic binoculars), you may also not be able to see Saturn's rings clearly.

**Comment by RacerBG** [comment]
> 
> 
> The "halo glow" vanishes automatically as you zoom in to get a clearer picture. Without this "halo" planets would vanish from view as they become too small. With too little magnification (3x plastic binoculars), you may also not be able to see Saturn's rings clearly.

Yes, I know that. If disabling it it's not a viable option then partially removing it at higher zoom levels would be fine too. The pictures which I created represent something around 50x-70x zoom compared to real life.

**Comment by axd1967** [comment]
See also https://github.com/Stellarium/stellarium/pull/1361

**Comment by LSAXYZ321** [comment]
www.stellarium-web.org via webbrowser do this way better, you can zoom more, like 5 times smaller FOV, so you can see Mars moons clearly. With Stellarium software they are just big glowing discs, instead of a single pixel when zooming out
